### ISSUE TITLE
Context, explicitly define

### ISSUE THREAD

**Issue by scottpeterson** [gryphonshafer/Quizzing-Rule-Book#56]
"Limited to five verses before or after the verse."   "verse" is not defined/specified/explicit in its usage here. 

In practice we do "five verses before or after the verse that the question starts on, not including the verse that the question starts on", so we end up with a 6 verse inclusive context, either direction.

Which is totally fine, it's not crystal clear from the current wording.

**Comment by scottpeterson** [comment]
I think is 75% solved since Gryphon's OCD pass rewrote this definition. The word "verse" has been purged.

The inclusive vs non-inclusive ambiguity still remains.

**Comment by aidanarthur** [comment]
What about changing "By default, context is limited to 5 verses before or after the reference" to "By default, context is limited to the reference of the question, the five verses preceding it, and the five verses following it"? That's a little clumsy to read, but I think it's more clear logically.
There are two other gaps I can see here: the debate over whether context spans chapter breaks and the obvious question of whether the second/third verse is part of the "reference" on an interrogative/MA.

**Comment by gryphonshafer** [comment]
I think that's an improvement, but we'll need to define explicitly the "reference of the question". That's easy with Q2V for example, but harder for SITs. There's the idea that pronoun clarification must be within context but does not influence context, and we need to decide how much of a SITs answer or quote must be considered the reference of the question. Nothing insurmountable, but needs to be explicit and clear.

**Comment by aidanarthur** [comment]
"By default, context is limited to all the verses containing the question and answer (excluding pronoun clarifications), the five verses preceding them, and the five verses following them."
This is getting long :)
One fringe case that's still fuzzy is if the question is in Book 1:3 and the answer is in Book 1:5, meaning that Book 1:4 is not part of the context under this definition.

**Comment by scottpeterson** [comment]
You're seeing the issues with the current wording. There's no "anchor" or "starting" point.

**Comment by zctinker** [comment]
We had that same thought all the time :) 

While maybe not the cleanest, v4 would be covered by going down 5 verses from 3 and by going up 5 verses from 5. Even though context extends further than just 5 verses down from verse 3 since it goes down 5 verses from verse 5. 

Example: Question is in v13 answer is in v15

True Context = 13-5 = v8     &      15+5 = v20
Relative context is 13-5 & 15-5 = v8 & v10     &     13+5 & 15+5 = v18 & v20

**Comment by scottpeterson** [comment]
I don't like question+answer structure being able to change "total verses in context"

**Comment by zctinker** [comment]
True, is a scenario like this 13 verses are in context while in a single verse Q&A only 11 verses are considered in context.

**Comment by scottpeterson** [comment]
Correct. To me, if context is defined as "5 verses up or down inclusive" then total verses in context should always be 11. But you are definitely making compromises either way. 

You could have a Question+ Answer from v6, but a pronoun clarification from v1. And if the starting point is defined as where the question starts, then the quizzer wouldn't be allowed to go any further back than v1. 

Which is probably fine.

**Comment by scottpeterson** [comment]
This feeds into Issue #38 somewhat.  Looking at the definition and implementation issues surrounding context, and the compromises that have to be made. Getting rid of context on INT, MA, SIT makes a lot of sense to me (and to Gryphon). The quizzer still can't give incorrect information, which, in my unscientific data is MOST of the reasons we're calling quizzers incorrect (other than not getting all required information).  It's pretty rare for a quizzer to (1) go out of context, (2) never say anything incorrect, (3) come back into context and get it right. Ditching context for INT, MA, SIT makes things a lot simpler.   We'd still need context on CVR/Q/F (the verses in question), and CR (either 5 verses up/down inclusive, or just the entire chapter).

**Comment by zctinker** [comment]
In my 7 years as a QM I can count on my fingers the number of times a quizzer went out of context without saying a wrong answer. If they can go out of context and still be consider not incorrect (but I won't say that they are correct), then that question many times should have been written as a Reference question (though of course not always) if they are clearly out of context, let them say something wrong before ruling them incorrect. 99% of all context things I have considered the quizzer answers for me by saying something else completely wrong. 

I like your above approach.

**Comment by kclimenhaga** [comment]
I think explicitly defining what the reference point is for context is a good idea. We do need to define whether context can go across chapters (I would say no). 

In terms of getting rid of context for those question types, I am not in favour. One issue I see arising would be quizzers intentionally pre-jumping and trying to quote long passages (for example, on a question like "Blessed are the peacemakers why?" in Matthew 5, a quizzer could jump on "Blessed" and try to quote all of verses 3-11). Another issue would be with situation questions. For who said it, quizzers could say "Paul" if the answer was "Saul" or vice versa. Would we consider this answer correct since both names refer to the same person? Currently, there is nothing that limits the answer a situation question to the speaker tag immediately preceding the quotation, nor do I think there should be since there are occasions when this is not possible.

**Comment by scottpeterson** [comment]
I am completely fine with a quizzer able to quote verses 3-11 and be counted correct. 

And currently a quizzer can be IN context, yet still give incorrect information. Eliminating context doesn't change that. 

For the "Paul/Saul" scenario, whether we'd call both acceptable (or at least not call one incorrect) if both are in context, I don't think one being out of context should change the potential rulings. 

In sum, the bulk of my rulings where a quizzer is counted incorrect rely on (a) Time Elapsing,  (b) giving incorrect information. Calling a quizzer wrong purely for going out of context happens very rarely. (giving wrong reference on a prejump is perhaps the most common one). 

The current language about a single word not taking you out of context, plus the undefined nature of what "takes you out of context" makes it problematic to invoke in all except the most obvious circumstances.

**Comment by scottpeterson** [comment]
In this case, I'd probably consider Paul/Saul as pretty interchangeable, since they refer to the same person. 

And if, because each name refers to a pretty different "state" of that single person, we think that a specific one should be required, both occurring in the same context is an irrelevant factor. 

If we think they're different enough, one occurring a verse away shouldn't change the ruling.

**Comment by scottpeterson** [comment]
I keep distilling my language. 

When evaluating content that a quizzer is saying, the text being in context doesn't make the text itself any "less incorrect" per se. 

A quizzer saying something that's in context only means they can't be counted incorrect solely for going out of context. 

Thus, when considering whether we want to evaluate context for INT/MA, the only impact will be on ability to count a quizzer incorrect solely for going out of context.  Everything else remains unchanged.

**Comment by jswingle** [comment]
I think it is reasonable for context to span across chapters, simply because the chapter breaks are often arbitrary. E.g. Hebrews 12:1 is very much in the same continuation of thought as Hebrews 11:40. Of course, the verse breaks are ALSO arbitrary, so that argument may not be fully consistent.

As currently written, I've always interpreted the rulebook as allowing context to go between chapters, because it never says it _doesn't._ But I think this would be good to clarify. I don't have an extremely strong opinion on which way we go, but my slight preference would be to allow it to go between chapters.

**Comment by ZacharyTinker** [comment]
Just because there are a couple of issues related to context, I want to tag them all here. 

#120 #116

**Comment by ZacharyTinker** [comment]
My personal preference is to eliminate context for INT, MA, and SIT questions. If a quizzer can keep on talking without saying anything considered incorrect, then my opinion would be to let them do just that. Keep talking and work it out, without saying anything incorrect.

But an alternative way of looking at context could be to define it as something larger. I know they aren't inspired, but we all have the same section headers in our official quiz books. It's uniform for everyone. So, what if context was within the section. This can cross chapters, this can cross 20 verses, or it can be just 5 verses.

Rarely are quizzers quoting material outside of a section, unless they are way off and are giving incorrect information.

**Comment by gryphonshafer** [comment]
Section context, which would give me all sorts of nostalgia feelings of days long past, is probably not a good idea because there are things like James 1:1 that blow it up...

**Comment by jttower** [comment]
This relates to my comment in #120 . I won't repeat it all here, so go there to read a specific example from IBQ last year.  It seems to me that the intent of the rule is to see if the quizzer is in the right section, in context of the narrative of the verses, not to call them incorrect if they are 6 verses away instead of only 5, unless of course it is a CVR or Q/F question. Therefore I agree (or at least I think I agree) that dropping the context requirement for INT/SIT/MA would help with this issue, possibly with a clarification about different titles as mentioned in #120 .

**Comment by jswingle** [comment]
+1 for eliminating context rules. My hesitation would be that I think this could create a LOT of rulebook bugs in various places. There needs to be plenty of deliberation on this one, and it needs to be considered in light of the rest of the rulebook.

e.g. We absolutely don't want elimination of context rules to lead to quizzers madly quoting a bunch of incorrect stuff and the QM counting them right because they eventually said the right thing and the QM couldn't keep up with all the incorrect stuff being said. There have to be _other_ well-written rules about what makes a quizzer "incorrect" which are compatible with the context rules being gone.

**Comment by JoshJetto** [comment]
I agree with Jeremy in regard to context crossing over chapters for the reason that Jeremy mentioned.
I am not in agreement with eliminating context on all of the question types to which the 5-verse context applies. The old rule from the 1990s that was on the books (according to my recollection) was 5 verses or the paragraph, whichever was greater. Although the verse numbers are not inspired, we are not asking in this case for quizzers to know the verse numbers in order to be called correct, but rather to be in the right area of the text - and I think we have to set limits on what that area is. I would be in favor of adding the "section" to the context. I also think that where a question and answer spans more than one verse that context should be the five verses before and after the entire question and answer block. 
The reason I am not in favor of eliminating context for INT/MA/SIT (and CR) questions is that it eliminates many of the risks of pre-jumping. If there is not a fairly tight context (5 verses and/or section), then a quizzer can jump at a point in the question where there are multiple options throughout the material, and simply run through the list of those options until they arrive at the right one and be called correct. Eliminating context would create a somewhat nonsensical jumping situation (and answering situation), where quizzers jump even earlier than they already do and just bounce around with attempted answers that are not incorrect information but are from all over the text. Not only would this be difficult for a quizmaster and answer judge to adjudicate - to know if there was incorrect information in the attempts (unless they also have all the material memorized), there would be very few scenarios in which we could call a quizzer incorrect before the 30 second timer had elapsed (exacerbating the timer issue), since, in the absence of a context rule, we would have to let them keep trying until time ran out.
In summary, I am in favor of clarifying the context as the five verses before and five verses after the entire question and answer block. I am in favor of adding "section" to what constitutes context and a "whichever is greater" clause with respect to five verses before and after or section context. I am in favor of context crossing chapter lines. And I am strongly opposed to the elimination of context altogether in INT/MA/SIT/CR questions.

**Comment by jswingle** [comment]
> I would be in favor of adding the "section" to the context.

This is an interesting proposal I hadn't thought of. A "whichever is greater, section or 5 verses" could work well. I think the "paragraph" rule that we used to have doesn't make much sense because the narrative material has frequent paragraph breaks for dialogue.

**Comment by gryphonshafer** [comment]
CBQZ uses the paragraph symbol to mean section. I love this section talk because the tech is ahead of the rule (for once).

**Comment by jswingle** [comment]
One thing that I'm not sure has been mentioned is that it's actually pretty impressive if a quizzer can jump on a vague phrase which happens to exist in 3 different contexts and just rattle off all the answers until they're counted correct. This is the sort of intense, high-level study that we _want_ to find ways to reward in quizzing. I think INT jumping could get VERY interesting, and there could be some real trailblazer quizzers who clean house with them because of excellent preparation and study. This is something I'd love to see.

On the other hand, I think removing context would make MA and SIT questions way too easy. A quizzer who can quote quickly and studies an MA/SIT list would have close to a 100% accuracy ratio on just 1-1.5 syllables. It would still require a lot of preparation, but it would also lead to a jump speed that I think we don't want to encourage.

And while I don't think CR questions would become "too easy," I do think it would sort of be a pity if CR questions on short chapters ended up just being "jump and quote as much as you can" and the entire strategy became speedquoting and figuring out the best places to make a quick stop so that the QM can ask you for your question. I think this would be a scenario we don't want on CRs, especially from the officials' perspective.

In sum: I'm a bit torn and I would like to continue the discussion for a long time on context rules before going in a particular direction with it. Right now, I see a lot of advantages and disadvantages to various approaches, and no clear solution. I just know that the current rule isn't great.

**Comment by scottpeterson** [comment]
While I usually focus mostly at the Internationals level, in this case, I don't think cons at that level of competition should nuke this idea.  (I agree with the Internationals level cons you listed Jeremy).  I love this idea of ditching Context for all MA/INT/SIT questions.   Ruling becomes much simpler.  We will reward quizzers that know more material.   

I still think you gotta keep Context on Q/F/CR/CVR.

**Comment by JoshJetto** [comment]
Since it's been awhile, I would refer back to my February 16 comment on this thread in response to Jeremy's most recent comment. I am very much not in favor of eliminating the context rules, though in that response, I outline modifications to it that would make it more clear and potentially slightly more expansive.
